integrations/aws/.port/spec.yaml

```diff
@@ -0,0 +1,29 @@
+type: aws
+description: AWS integration for Port Ocean
+icon: AWS
+features:
+  - type: exporter
+    section: Cloud Providers
+    resources:
+      - kind: EC2
+configurations:
+  - name: awsAccessKeyId
+    required: true
+    type: string
+    sensitive: true
+  - name: awsSecretAccessKey
+    required: true
+    type: string
+    sensitive: true
+  - name: organizationRoleArn
+    required: false
+    type: string
+    sensitive: true
+  - name: accountReadRoleName
+    required: false
+    type: string
+    sensitive: true
+  - name: awsApiKey
+    required: false
+    type: string
+    sensitive: true
```

Comment:
All of these are applied like i asked for other integrations
No need for the customer to supply those when he is running over his own account
utilize boto3 and what he expects as for these credentials